env/env_test.go

```diff
@@ -63,3 +65,64 @@ func TestGetEnvUnsupported(t *testing.T) {
 		assert.Equal(t, 1, env.GetEnv[[]int]("MY_INT_ARRAY", env.WithDefault("[1]")))
 	})
 }
+
+func TestLoadEnvsFromFile(t *testing.T) {
+	tempDir := t.TempDir()
+
+	tests := []struct {
```

Comment:
Table tests don’t always make test simpler. In this case, they actually make the test harder to read.

It's better to split it into two separate test cases with clear scenarios:

**Scenario 1:**  
1. Write `FOO=bar` into a file  
2. Call the function  
3. Check that the `FOO` environment variable is set to `bar`

**Scenario 2:**  
- Call the function with a non-existent file like `file-not-exists.env`  
- It should panic: use `assert.Panics(...)`